run/GCClassic/HEMCO_Config.rc.templates/HEMCO_Config.rc.carbon

```diff
@@ -1232,10 +1280,37 @@ ${RUNDIR_CO2_COPROD}
 #==============================================================================
 # --- Seasonal scaling factors ----
 #==============================================================================
-(((EDGARv6.or.GEPA
+(((GEPA.or.Scarpelli_Mexico.or.Scarpelli_Canada
 10 MANURE_SF $ROOT/CH4/v2017-10/Seasonal_SF/EMICH4_Manure_ScalingFactors.WithClimatology.nc  sf_ch4 2008-2016/1-12/1/0 C xy 1 1
 11 RICE_SF   $ROOT/CH4/v2017-10/Seasonal_SF/EMICH4_Rice_ScalingFactors.SetMissing.nc         sf_ch4 2012/1-12/1/0  C xy 1 1
-)))EDGARv6.or.GEPA
+)))GEPA.or.Scarpelli_Mexico.or.Scarpelli_Canada
+
+(((EDGARv7
```

Comment:
Same question as for HEMCO_Config.rc.CH4 above (does EDGARv6 have a scale factor block)

Reply:
See comment above.